src/formio/components/FileField.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import {Canvas, Meta, Props, Story} from '@storybook/addon-docs';
+
+import {FileField} from './FileField';
+import {FormDecorator, FormioComponent} from './story-util';
+import {
+  BASE_URL,
+  ENDPOINT,
+  mockFileUploadError,
+  mockFileUploadPost,
+} from '/src/mocks/FileUploadMocks';
+
+export const FILETYPES = [
+  'all',
+  'heic',
+  'png',
+  'jpg',
+  'pdf',
+  'xls',
+  'xlsx',
+  'csv',
+  'txt',
+  'doc',
+  'docx',
+  'open_office',
+  'zip',
+  'rar',
+  'tar',
+  'msg',
+  'dwg',
+];
```

Comment:
the component itself requires mime types, no? instead of extensions

Reply:
The component itself takes mime types, not extensions, but the user who creates forms selects mime types only indirectly, viz. by selecting file extensions. If the purpose is purely illustrative, then it doesn't make sense to list mime types.